.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+  - "4.2"
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3 # give xvfb some time to start
+  - npm install -g selenium-standalone
+  - selenium-standalone install
+  - selenium-standalone start > /dev/null 2>&1 &
```

Comment:
Instead of rewriting the server startup, please use the `protractor` + `npm run webdriver-manager-start` combination that's already written/documented. This will lead to more consistency in our repo:

https://github.com/twolfson/google-music.js/tree/3.3.2#testing

https://github.com/twolfson/google-music.js/blob/3.3.2/package.json#L37

https://github.com/twolfson/google-music.js/blob/3.3.2/package.json#L54

```yml
# protractor already installed via `install` step
- npm run webdriver-manager-start &> /dev/null
- sleep 3
```